src/test/java/org/sonatype/nexus/repository/composer/internal/ComposerJsonExtractorTest.java

```diff
@@ -58,6 +58,20 @@ public void extractInfoFromZipballWithJson() throws Exception {
     assertEquals(expected, actual, true);
   }
 
+  @Test
+  public void extractInfoFromZipballWithJsonComposerArchived() throws Exception {
+    String expected;
+    try (InputStream in = getClass().getResourceAsStream("extractInfoFromZipballWithJsonComposerArchived.composer.json")) {
+      expected = CharStreams.toString(new InputStreamReader(in, StandardCharsets.UTF_8));
+    }
+    String actual;
+    try (InputStream in = getClass().getResourceAsStream("extractInfoFromTipBallWithJsonComposerArchived.zip")) {
```

Comment:
Minor: Change "TipBall" to "Zipball" here and in the associated file. I'm guessing one's just an accident from the T and the Z being side by side on QWERTZ layouts, and the other change is just to be consistent with our naming conventions elsewhere.